var/spack/repos/builtin/packages/py-fuzzywuzzy/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyFuzzywuzzy(PythonPackage):
+    """Fuzzy string matching in python."""
+
+    homepage = "https://github.com/seatgeek/fuzzywuzzy"
+    pypi     = "fuzzywuzzy/fuzzywuzzy-0.18.0.tar.gz"
+
+    version('0.18.0', sha256='45016e92264780e58972dca1b3d939ac864b78437422beecebb3095f8efd00e8')
+
+    variant('speedup', default=False, description='Provide a 4-10x speedup')
```

Comment:
Would it be worth to either rename or have it always required?

Reply:
For the name, I just use the name given in `extra-requires`. This is the same name used when you run `pip install fuzzywuzzy[speedup]`.

For the default, I usually default to disabling extra dependencies. I don't know if `py-python-levenshtein` takes that long to build, so I'm not fixed to this default.

Reply:
Ok, I thought the variant name was yours :laughing: If it comes from upstream I think it's "fine"